src/nyc_trees/apps/survey/migrations/0027_recreate_group_territory_20150526_1129.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.utils.timezone import now
+
+
+def forwards(apps, schema_editor):
+    Territory = apps.get_model('survey', 'Territory')
+    BlockfaceReservation = apps.get_model('survey', 'BlockfaceReservation')
+    Blockface = apps.get_model('survey', 'Blockface')
+    Group = apps.get_model('core', 'Group')
+
+    possible_groups = Group.objects.filter(border__isnull=False)
+
+    nyc_parks = Group.objects.filter(slug='nyc-parks')
+    nyc_parks = nyc_parks[0] if nyc_parks else None
+
+    reserved_ids = BlockfaceReservation.objects.values_list('blockface_id', flat=True)
```

Comment:
Without this, if an individual mapper mapped a block in nobody's territory it would appear in `orphaned_blocks`.